line-auth/functions/index.js

```diff
@@ -85,33 +85,34 @@ function getFirebaseUser(lineMid, lineAccessToken) {
  *
  * @returns {Promise<string>} The Firebase custom auth token in a promise.
  */
-function verifyLineToken(lineAccessToken) {
+ function verifyLineToken(lineAccessToken) {
   // Send request to LINE server for access token verification
   const verifyTokenOptions = generateLineApiRequest('https://api.line.me/v1/oauth/verify', lineAccessToken);
 
   // STEP 1: Verify with LINE server that a LINE access token is valid
   return rp(verifyTokenOptions)
-    .then(response => {
+  .then(response => {
```

Comment:
`.then`s indent since they are a line continuation. Multi-line `then`s continue with the closing bracket of the previous statement.

```
rp(verifyTokenOptions)
  .then(response => {
  }).then(userRecord => {
  }).then(token => {
  })
```